actions/new.action.ts

```diff
@@ -213,13 +224,13 @@ const askForMissingInformation = async (inputs: Input[], options: Input[]) => {
   }
 
   const prismaInput = getPrismaInput(options);
-  if (!prismaInput!.value) {
+  if (prismaInput && prismaInput.value === undefined) {
```

Comment:
this condition has been reused mutitple times, can be abstract this out as a function call and optimise the code using loops and function calls if possible?